components/home/homepage-menu.tsx

```diff
@@ -20,7 +20,11 @@ export function HomePageMenu() {
             })}
           </div>
           <Link href={item.href}>
-            <a className="text-gray-900 font-semibold text-xs mt-3 helper-link-cover">
+            <a
+              className="text-gray-900 font-semibold text-xs mt-3 helper-link-cover"
+              rel={item.external ? "noopener noreferrer nofollow" : undefined}
+              target={item.external ? "_blank" : undefined}
+            >
```

Comment:
It can not like this, can not use next `Link` to open new tab.
You should create different tag.

Reply:
ahh yap 😭, I forget to re-check.

should we just <a> for implement it?